standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -3911,7 +4166,7 @@ public List<String> get_table_names_by_filter(
   }
 
   private Partition append_partition_common(RawStore ms, String catName, String dbName,
-                                            String tableName, List<String> part_vals,
+                                            String tableName, List<String> part_vals, boolean skipFSWrites,
```

Comment:
typically, we add new arguments to the end of the current list. This is a private method, so not a big different, but would be nice to make the new argument the last one.